Automattic/wp-calypso
# Conversations: filter out pingbacks/trackbacks

elasticsearch already doesn't give us pingbacks and trackbacks.
we need to make sure we don't fetch any from the API in conversations.

fixes: https://github.com/Automattic/wp-calypso/issues/19236

**to test**
you need to make sure that you no longer see any trackbacks/pingbacks in the convos stream.  best way of doing this is:
1. go to `wordpress.com/read/conversations`. notice the first followed convo that has a ping
2. open this branch and scroll to the same convo.  ensure the ping is no longer there.


**note**
I uncovered a different issue while pursuing this.  the comment count is off in cases where we couldn't retrieve all comments because either:
1. some are pingbacks/trackbacks
2. it was deleted/there was an error in retrieving it

the solution may be as simple as verifying if there are any more comments to pull, and if there aren't then assume what we have in redux is the correct count.
edit: opened an issue https://github.com/Automattic/wp-calypso/issues/20273

Fixes #20273.
